worlds/sc2/items.py

```diff
@@ -127,6 +128,8 @@ def get_full_item_list():
 SC2WOL_ITEM_ID_OFFSET = 1000
 SC2HOTS_ITEM_ID_OFFSET = SC2WOL_ITEM_ID_OFFSET + 1000
 SC2LOTV_ITEM_ID_OFFSET = SC2HOTS_ITEM_ID_OFFSET + 1000
+SC2_KEY_ITEM_ID_OFFSET = SC2LOTV_ITEM_ID_OFFSET + 1000
+SC2_KEY_ITEM_SECTION_SIZE = 1000 # Reserve this many IDs for missions, layouts, campaigns, and generic keys each
```

Comment:
(super minor, optional) PEP8 requests 2 spaces before the `#` for inline comments.

Reply:
Honestly if I saw that I would delete the second space as a bug lol, I'll switch this to its own line.